### ISSUE TITLE
Vertical CalendarList renderHeader is not producing any output

### ISSUE THREAD

**Issue by Singhmansingh** [wix/react-native-calendars#1216]
Please make our job easier by filling this template out to completion. If you're requesting a feature instead of reporting a bug, please feel free to skip the Environment and Reproducible Demo sections.

## Description

Hi, ive been trying to have the header of each calendar in the calendarlist be a TouchableOpacity button, however the component doesn't seem to accept the renderHeader prop, and no changes occur.

## Expected Behavior
<img width="750" alt="calendarExpectedAsset 1@2x" src="https://user-images.githubusercontent.com/64917885/86166789-5d119f80-bae3-11ea-841a-03e4df70e45b.png">

## Observed Behavior

![IMG_E1336](https://user-images.githubusercontent.com/64917885/86166810-67339e00-bae3-11ea-940b-3c57d8c479cd.JPG)

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-calendars`: `-- react-native-calendars@1.300.0
* `npm ls react-native`: `-- react-native@0.61.4

Also specify:

1. Phone/emulator/simulator & version: expo client version 2.16.0.108985

## Reproducible Demo
`  <CalendarList
              calendarWidth={Layout.window.width}
              pastScrollRange={3}
              futureScrollRange={6}
              renderHeader={(date) => {return (<TouchableOpacity onPress = {() => console.log(date)}><Text>HELLO WORLD</Text></TouchableOpacity>)}}
              />`

Issues that come with minimal repro's are resolved much more quickly than issues where a maintainer has to reproduce themselves.

**Comment by emilisb** [comment]
I tested your code in our example app (example/src/screens/calendarsList.js) and it works for me. Can you please provide more details?

![Simulator Screen Shot - iPhone 11 Pro - 2020-07-01 at 10 17 33](https://user-images.githubusercontent.com/13116854/86214690-2344a580-bb84-11ea-8e9a-965944da4d9b.png)

**Comment by Singhmansingh** [comment]
Thanks for the fast reply. I’m Not sure what I need to provide As i am a
novice, but My list is Located in a nested navigator, using a function that
extends react.component. The app is built on the expo tabs template as
well. The calendarlist itself is placed in a view with no styling. When i
use the calendarlist itself, typing “renderheader” doesn't bring up any
autofill Prop with a description. When i Cntrl-clicked into the component,
it Took me to the index.d.ts file with all of the exported props (for all 3
calendar types), but renderHeader was not one of them.

**Comment by emilisb** [comment]
@Singhmansingh I took your exact code (except for changing `calendarWidth` to a numeric value) and it works. Can you create an example repo and post it on Github?

<img width="1034" alt="Screenshot 2020-07-02 at 15 48 06" src="https://user-images.githubusercontent.com/13116854/86360830-98df6d00-bc7b-11ea-920f-597fd762eae9.png">

**Comment by Singhmansingh** [comment]
> @Singhmansingh I took your exact code (except for changing `calendarWidth` to a numeric value) and it works. Can you create an example repo and post it on Github?
> 
> <img alt="Screenshot 2020-07-02 at 15 48 06" width="1034" src="https://user-images.githubusercontent.com/13116854/86360830-98df6d00-bc7b-11ea-920f-597fd762eae9.png">

Im not able to produce a repo right now, but as soon as i am able I will.

**Comment by Singhmansingh** [comment]
https://github.com/singhmansingh/calendarlisttest

i tested it out on an old version of the expo template (the same one my app is built on), and I got it to work, so I will need to figure out why it was not working in my main app (the version of react-native-calendars and react native are the same in both instances). 

however, I tried to create the example with the new expo-template-tabs, which has changed the file structure, and I found an error with the render header as seen in the image below. I am not familiar with .tsx files, so I am not sure if  its meaningful or if I just did not install the template properly. but thanks for your help with this issue.

![calendarList error](https://user-images.githubusercontent.com/64917885/86415174-36469b00-bc94-11ea-967b-83a97a7f2668.png)
